From 7bcc1269ca9e629b4f7782e769b07330e5937956 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Thu, 3 Apr 2014 17:32:52 +0200
Subject: [PATCH] Update user-visible copyright information

Years in copyright notices in the about dialog and man pages is at most
2012, let's set it to 2014
---
 man/remote-viewer.pod     | 2 +-
 man/virt-viewer.pod       | 2 +-
 src/virt-viewer-about.xml | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/man/remote-viewer.pod b/man/remote-viewer.pod
index 6773c3b..e35d281 100644
--- a/man/remote-viewer.pod
+++ b/man/remote-viewer.pod
@@ -283,7 +283,7 @@ Report bugs to the mailing list C<http://www.redhat.com/mailman/listinfo/virt-to
 
 =head1 COPYRIGHT
 
-Copyright (C) 2012 Red Hat, Inc., and various contributors.
+Copyright (C) 2012-2014 Red Hat, Inc., and various contributors.
 This is free software. You may redistribute copies of it under the terms of the GNU General
 Public License C<http://www.gnu.org/licenses/gpl.html>. There is NO WARRANTY, to the extent
 permitted by law.
diff --git a/man/virt-viewer.pod b/man/virt-viewer.pod
index 963d7bc..5b3d6e6 100644
--- a/man/virt-viewer.pod
+++ b/man/virt-viewer.pod
@@ -148,7 +148,7 @@ Report bugs to the mailing list C<http://www.redhat.com/mailman/listinfo/virt-to
 
 =head1 COPYRIGHT
 
-Copyright (C) 2007-2008 Red Hat, Inc., and various contributors.
+Copyright (C) 2007-2014 Red Hat, Inc., and various contributors.
 This is free software. You may redistribute copies of it under the terms of the GNU General
 Public License C<http://www.gnu.org/licenses/gpl.html>. There is NO WARRANTY, to the extent
 permitted by law.
diff --git a/src/virt-viewer-about.xml b/src/virt-viewer-about.xml
index e9ae52a..16db3c2 100644
--- a/src/virt-viewer-about.xml
+++ b/src/virt-viewer-about.xml
@@ -14,7 +14,7 @@
     <property name="skip_pager_hint">True</property>
     <property name="program_name">Virtual Machine Viewer</property>
     <property name="copyright" translatable="yes">Copyright (C) 2007-2012 Daniel P. Berrange
-Copyright (C) 2007-2012 Red Hat, Inc.</property>
+Copyright (C) 2007-2014 Red Hat, Inc.</property>
     <property name="comments" translatable="yes">A remote desktop client built with GTK-VNC, SPICE-GTK and libvirt</property>
     <property name="website">http://virt-manager.org/</property>
     <property name="website_label" translatable="yes">virt-manager.org</property>
